enterprise/admin/forms.py

```diff
@@ -14,10 +14,12 @@
 from django.utils.translation import ugettext as _
 
 from enterprise import utils
-from enterprise.admin.utils import ValidationMessages, email_or_username__to__email, validate_email_to_link
-from enterprise.course_catalog_api import get_all_catalogs
+from enterprise.admin.utils import (ProgramStatuses, ValidationMessages, email_or_username__to__email,
```

Comment:
@mattdrayer if you mean `from admin.utils import *` - probably never as pylint will complain. If you mean just `import admin.utils` and prefixing utils members - probably someday. But the better approach might be to split utils into more granular modules or (to the opposite) impose some classes to contain logically related utils.